runelite-client/src/main/java/net/runelite/client/plugins/config/PluginHubPanel.java

```diff
@@ -186,14 +186,14 @@
 			JButton help = new JButton(HELP_ICON);
 			help.setRolloverIcon(HELP_ICON_HOVER);
 			SwingUtil.removeButtonDecorations(help);
-			help.setToolTipText("Help");
 			help.setBorder(null);
 			if (manifest.getSupport() == null)
 			{
 				help.setVisible(false);
 			}
 			else
 			{
+				help.setToolTipText("Open help to: " + manifest.getSupport().toString());
```

Comment:
The "to" sounds awkward. Just `Open help: https://github.com/user/repo` is better in my opinion.

```suggestion
				help.setToolTipText("Open help: " + manifest.getSupport().toString());
```